packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.TypeScript.MSBuild" version="4.7.4" targetFramework="net40" developmentDependency="true" />
```

Comment:
Do we need to be including the full nuget package in the repo? Can't the build resolve package dependencies?

Reply:
I like the idea of adding the npm install ejs to the build itself so we don't need to copy a dependency module within our own repo. Let me see what we can modify.

Reply:
Would this impact local builds? Don't we still need the package config file to inform my local build of the needed packages to restore?

Reply:
Looks like we can safely remove it and local building works fine. the solution file wants to run a nuget restore for that typescript package. But it shouldn't be required for our automation. Just for devs who prefer to run in Visual Studio rather than command line.